<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>668</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华文楷体</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>唐义军的信安综合课设</string>
  </property>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encry_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(55, 85, 79)</string>
   </property>
   <property name="text">
    <string>加密文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decry_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(181, 255, 176)</string>
   </property>
   <property name="text">
    <string>解密文件</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_show">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>671</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="setting">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输出文件名称，默认output.xxx</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="AES_Check">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>AES</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_input">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>输入密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="empty_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空消息</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="SM4_Check">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SM4</string>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
